src/main/java/seedu/brokeMan/command/ExitCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.brokeMan.command;
+
+import seedu.brokeMan.save.SaveBudget;
+import seedu.brokeMan.save.SaveExpense;
+import seedu.brokeMan.save.SaveIncome;
+import seedu.brokeMan.ui.Ui;
+
+import static seedu.brokeMan.budget.Budget.budgetEachMonth;
+import static seedu.brokeMan.entry.expense.ExpenseList.expenseList;
+import static seedu.brokeMan.entry.income.IncomeList.incomeList;
+
+public class ExitCommand extends Command {
+    public static final String COMMAND_WORD = "exit";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": exits the program\n" +
+            "|  Example: " + COMMAND_WORD;
+    public void execute() {
+        Ui.showToUserWithLineBreak("Exiting program...", "");
+        SaveExpense.writeFile(expenseList);
+        SaveIncome.writeFile(incomeList);
+        SaveBudget.writeFile(budgetEachMonth);
```

Comment:
I noticed that you only save your data to the file at the end of program. This could be an issue if your program was somehow disrupted during usage and exits unexpectedly. You could consider changing this approach. 👍